Python/Distance-Calculator-App/main.py

```diff
@@ -0,0 +1,22 @@
+from geopy.geocoders import Nominatim
+from geopy import distance
+import argparse
+def location():
```

Comment:
@vybhav72954 Thanks for reviewing my PR, can you please tell what to enter in docstring?

Reply:
here this [script](https://github.com/HarshCasper/Rotten-Scripts/blob/master/Python/Github_Traffic/github_traffic.py) has some good DocStrings.


In case you are wondering what docstring is, click [here](https://www.geeksforgeeks.org/python-docstrings/)


For a simpler explanation, [click here](https://github.com/HarshCasper/Rotten-Scripts/blob/master/STYLE_GUIDELINES.md)